products/rabbitmq.md

```diff
@@ -11,6 +11,11 @@ releaseDateColumn: true
 eolColumn: Community Support
 eoesColumn: Extended Commercial Support
 
+identifiers:
+-   repology: rabbitmq-c
+-   cpe: cpe:/a:rabbitmq-c_project:rabbitmq-c
+-   cpe: cpe:2.3:a:rabbitmq-c_project:rabbitmq-c
```

Comment:
```suggestion
-   repology: rabbitmq-server
-   cpe: cpe:/a:pivotal_software:rabbitmq
-   cpe: cpe:2.3:a:pivotal_software:rabbitmq
-   cpe: cpe:/a:vmware:rabbitmq
-   cpe: cpe:2.3:a:vmware:rabbitmq
```

rabbitmq-c seems to be a client : https://github.com/alanxz/rabbitmq-c. I would use this instead based on https://en.wikipedia.org/wiki/RabbitMQ.